Modix.Services/Core/MessageLogBehavior.cs

```diff
@@ -54,6 +62,44 @@ private async Task<bool> ShouldSkip(IGuild guild, IMessageChannel channel)
             return result;
         }
 
+        private async Task HandleMessageReceived(SocketMessage message)
+        {
+            Log.LogDebug("Handling message received event for message #{MessageId}.", message.Id);
+
+            if (!message.Content.StartsWith('!') &&
```

Comment:
Please don't hard-code this. Unless there's a legit way to retrieve the configured command prefix within this scope, just log it like any other message.

Reply:
Is there a way?

Reply:
I agree this should probably be a configurable thing, especially if we're already thinking about other guilds would use MODiX. If/when this gets promoted to a setting we should remove this.

Reply:
If you really wanna leave it in, put a TODO on it.